frontend/institution/managementMembersController.js

```diff
@@ -7,6 +7,8 @@
         RequestInvitationService, ProfileService) {
         var manageMemberCtrl = this;
 
+        var MAX_EMAILS = 1;
```

Comment:
Why is 1, the maximum number of emails? You should give a better name.

Reply:
Because the view is not ready to support more than that, and that's because de design is not ready yet.